src/clients/web3/useAuth/index.ts

```diff
@@ -9,7 +9,7 @@ import {
   UserRejectedRequestError as UserRejectedRequestErrorWalletConnect,
   WalletConnectConnector,
 } from '@web3-react/walletconnect-connector';
-import toast from 'components/Basic/Toast';
+import toast from 'components/v2/Toast';
```

Comment:
Outside of the component directory it should be imported directly from components
import { toast } from 'components';

When we deprecate the other components and get rid of the v2 directory the only paths we should have to change are the `index.ts` in components